internal/work/dbus_test.go

```diff
@@ -43,6 +43,12 @@ func TestScrubName(t *testing.T) {
 			want:        "alpha_bravo_charlie",
 			wantError:   cmpopts.AnyError,
 		},
+		{
+			description: "old rhc-worker-playbook",
+			input:       "rhc-worker-playbook",
+			want:        "rhc_worker_playbook",
+			wantError:   cmpopts.AnyError,
+		},
```

Comment:
How is this test case any different from the other cases, specifically "multiple hyphens"?

Reply:
Yeah, I added this only for being sure that it works as expected for this worker. Removed this change in the latest version of this PR.